### ISSUE TITLE
Dependabot + changesets Bot

### ISSUE THREAD

**Issue by aaronmgdr** [celo-org/reusable-workflows#85]
Write a bot which for dependabot prs commits a changeset file with the description from dependabot.

**Comment by arthurgousset** [comment]
Is this still useful @aaronmgdr ? Since we moved to renovate, this might not be necessary anymore.

**Comment by arthurgousset** [comment]
For example [here](https://github.com/celo-org/developer-tooling/blob/master/packages/sdk/contractkit/CHANGELOG.md#patch-changes): 

> Updated dependencies [[6b2e34c](https://github.com/celo-org/developer-tooling/commit/6b2e34c973290da221aaabdc2bf4c6654ef9f99c)]:
> 
> - @celo/utils@6.0.0
> - @celo/wallet-local@5.1.2
> - @celo/connect@5.1.2

**Comment by aaronmgdr** [comment]
yes because since renovate doesnt create changeset files when we version/publish if the only updates to a package are from dep updates thru renovate it will not know to bump version. 

other but less important is that the dep upgrade to not add to changlogs
